packages/calcite-components/src/components/navigation-logo/navigation-logo.tsx

```diff
@@ -107,34 +113,45 @@ export class CalciteNavigationLogo implements LoadableComponent {
     return <calcite-icon class={CSS.icon} flipRtl={this.iconFlipRtl} icon={this.icon} scale="l" />;
   }
 
+  renderHeaderContent(): VNode {
+    const { heading, headingLevel, description } = this;
+    const headingNode = heading ? (
+      <Heading class={CSS.heading} level={headingLevel}>
+        <span
+          aria-label={this.heading}
```

Comment:
I don't think we need an aria-label here. The heading element should already provide enough information.